cmake/printInfo.cmake

```diff
@@ -30,7 +30,11 @@ if(NOT DEFINED BUILD_SHARED_LIBS)
 else()
     message( STATUS "BUILD_SHARED_LIBS =     ${BUILD_SHARED_LIBS}" )
 endif(NOT DEFINED BUILD_SHARED_LIBS)
+message( STATUS "LIBXML2_XMLLINT_EXECUTABLE = ${LIBXML2_XMLLINT_EXECUTABLE}" )
 message( STATUS "BUILD_TESTS =           ${BUILD_TESTS}" )
+if(BUILD_TESTS)
```

Comment:
This should be `BUILD_TESTING` as that is consistent with the variable ctest defines, but I guess its already been there for awhile.

Reply:
Will look into this later. Probably needs to  support both and be deprecated in a later version.